synapse/storage/databases/main/receipts.py

```diff
@@ -288,7 +290,7 @@ async def get_linearized_receipts_for_rooms(
 
     async def get_linearized_receipts_for_room(
         self, room_id: str, to_key: int, from_key: Optional[int] = None
-    ) -> List[dict]:
+    ) -> Sequence[dict]:
```

Comment:
```suggestion
    ) -> Sequence[JsonDict]:
```

for consistency with `_get_linearized_receipts_for_room` below

Reply:
done